bdk_keychain/src/keychain_txout_index.rs

```diff
@@ -85,7 +85,7 @@ impl<K: Clone + Ord + Debug> KeychainTxOutIndex<K> {
     /// Scan a single `TxOut` for a matching script pubkey.
     ///
     /// If it matches the index will store and index it.
-    pub fn scan_txout(&mut self, op: OutPoint, txout: &TxOut) {
+    pub fn scan_txout(&mut self, op: OutPoint, txout: &TxOut) -> Option<(K, u32)> {
```

Comment:
Is this change a necessary part of this PR?

Reply:
No, woops 😅